docs/_articles/en/v3.x/release-information.md

```diff
@@ -6,6 +6,39 @@ lang: en
 ### To update the plugin, please follow the linked [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in)
 
 
+## [v3.4.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.4.0) (08-17-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.3.0...v3.4.0)
+
+### Release Summary
+* Improvements to `eslint`:
+	- Execute `eslint` engine with babel parser to accommodate more modern javascript syntax
+	- Identify javascript target files that were analyzed by both `eslint` and `eslint-lwc`, and throw warning about the possibility of duplicate violations
+* Survey request banner fix
+* SFGE: Minor code fixes
+* Updates to local RetireJS Vulnerability Repository
+* PMD upgrade to 6.48.0
+
+**Closed issues:**
+
+- \[BUG\] UnexpectedException on v3 DFA on large ISV codebase [\#739](https://github.com/forcedotcom/sfdx-scanner/issues/739)
+- \[BUG\] stdout not in proper JSON format when --format json is used [\#771](https://github.com/forcedotcom/sfdx-scanner/issues/771)
+- apex.jorje.parser.impl.BaseApexLexer dedupe [\#779](https://github.com/forcedotcom/sfdx-scanner/issues/779)
+- \[BUG\] adding custom rule [\#788](https://github.com/forcedotcom/sfdx-scanner/issues/788)
+
+**Merged pull requests:**
+
+- @W-11446174@: changed eslint parser to @babel/eslint-parser [\#767](https://github.com/forcedotcom/sfdx-scanner/pull/767)
```

Comment:
@W-11446174@ CHANGE: We switched the eslint parser from X to `@babel/eslint-parser`.
@W-10778096@ NEW: Unresolvable parameters are now passed to database operations.
@W-10778096@ FIX: We made some minor edits to error messages.
@W-11494080@ FIX: The survey request is formatted properly.
@W-11476976@ FIX: `stripInaccessible()` is not blocked on custom values.
- @W-11446192@ NEW: A warning is emitted if any files are processed by both eslint and eslint-lwc.
- @W-11505293@ NEW: Skip info logs from dependency (Roopa, this needs more detail about what dependency?!)
- @W-10778080@ NEW: Unresolved Apex values in DML statements are handled.
- @W-11567651@ NEW: PMD v6.48.0 is available.

Reply:
Similar to above, we should be mindful of the titles we provide to PRs we create.